DESCRIPTION

```diff
@@ -1,7 +1,7 @@
 Type: Package
 Package: ospsuite
 Title: R package to manipulate OSPSuite Models
-Version: 10.0.0
+Version: 10.0.70

```

Comment:
@Yuri05 @IndrajeetPatil I checked what was the last version on appveyor so that we would get the expected version here...Not required once we automate the deploy